agent/exporter/src/main/java/com/microsoft/applicationinsights/agent/Exporter.java

```diff
@@ -118,16 +119,19 @@ private void export(SpanData span) {
             boolean isLog = removeAttributeBoolean(attributes, AI_INTERNAL_LOG);
             if (isLog) {
                 exportLogSpan(span, attributes);
-            } else if (!SpanId.isValid(span.getParentSpanId())) {
-                // TODO revisit this decision
-                // maybe user-generated telemetry?
-                // otherwise this top-level span won't show up in Performance blade
+            } else if ("spring-scheduling".equals(stdComponent) && !SpanId.isValid(span.getParentSpanId())) {
```

Comment:
Should i rename SpringScheduler to SpringScheduling for smoke test?  It's SpringSchedulingInstrumentation on the opentelemetry as well.

Reply:
sure, that seems more consistent 👍